Dockerfile.devkit

```diff
@@ -135,9 +135,8 @@ RUN if [ "${DOCKER_GID}" -ne "${GROUP_ID}" ]; then \
 # place it in /usr/share/ansible/collections, the container will be run with a different user
 RUN mkdir -p /usr/share/ansible/collections && ansible-galaxy collection install ansible.utils -p /usr/share/ansible/collections
 
-
 # hadolint ignore=DL3059
-RUN packer plugins install github.com/hashicorp/googlecompute ">=1.0.0" \
+RUN --mount=type=secret,id=githubtoken PACKER_GITHUB_API_TOKEN=$(cat /run/secrets/githubtoken) packer plugins install github.com/hashicorp/googlecompute ">=1.0.0" \
```

Comment:
where is `/run/secrets/githubtoken` created?

Reply:
it gets created by the secret mount when we pass in `--secret ` in our docker buildx build command